wikiHandler.js

```diff
@@ -177,5 +193,37 @@ async function findWikiData(auction_contents, server) {
     return resolved_wiki_data;
 }
 
+async function fetchImageUrl(itemName) {
```

Comment:
So wait, where do these images go? I don't actually see them anywhere...

Reply:
They go in the response to !watches and watch notifications.  If the item name doesn't match a url, it isn't displayed.  Figured I could improve that later.

Reply:
Oh, ok they show up when you do a WATCH. I was looking for them in the auction stream. 👍